bloxid/hashids.go

```diff
@@ -0,0 +1,108 @@
+package bloxid
+
+import (
+	"errors"
+
+	hashids "github.com/speps/go-hashids/v2"
+)
+
+const (
+	hashIDAllowedChar = "0123456789abcdef"
+
+	//the prefix needs to be uppercase so there are no collisions with chars in hashIDAllowedChar
+	hashIDPrefix = "HIDZ"
+
+	idTypeHashID = "hashid"
```

Comment:
to not confuse EntityType() with IDType(), let's just use different name, i.e. Scheme()
```suggestion
	idSchemeHashID = "hashid"
```